src/main/java/org/isf/admission/manager/AdmissionBrowserManager.java

```diff
@@ -308,6 +308,18 @@ public Patient deletePatientPhoto(int id) throws OHServiceException {
 	protected void validateAdmission(Admission admission, boolean insert) throws OHServiceException {
 		List<OHExceptionMessage> errors = new ArrayList<>();
 
+		Ward ward = admission.getWard();
+		if (ward == null) {
+			errors.add(new OHExceptionMessage(MessageBundle.getMessage("angal.admission.admissionwardcannotbeempty.msg")));
+			throw new OHDataValidationException(errors);
+		}
+
+		LocalDateTime dateIn = admission.getAdmDate();
+		if (dateIn == null) {
+			errors.add(new OHExceptionMessage(MessageBundle.getMessage("angal.admission.admissiondatecannotbeempty.msg")));
+			throw new OHDataValidationException(errors);
+		}
+
```

Comment:
Also in lines 314 and 320 should not be there the throw new, unless we want to stop immediately the validation at that point (makes sense).

Now that I think about it, it seems we miss a check for `patient == null` after line 326, is it?

Reply:
I have changes ready to add the `patient` check and rework the section slightly.   Waiting on responses to other comments before comitting.

Reply:
code committed